File: 7-kyu/convert-the-score/python/solution.py
def scoreboard(strng):
    first, second = None, None
    scores = ["nil", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"]
    for word in strng.split():
        if word in scores:
            if first == None:
                first = scores.index(word)
            elif second == None:
                second = scores.index(word)
    return [first, second]

File: 7-kyu/convert-the-score/javascript/solution.js
const scoreboard = str => {
  const score = [];
  const callouts = ["nil", "one", "two", "three", "four", "five", "six", "seven", "eight", "nine"];
  for (const word of str.split(" ")) {
    if (callouts.includes(word)) {
      const idx = callouts.indexOf(word);
      score.push(idx);
    }
  }
  return score;
};
